core/Pos/Arbitrary/Crypto.hs

```diff
@@ -79,6 +85,29 @@ instance Nonrepeating PublicKey where
 instance Nonrepeating SecretKey where
     nonrepeating n = map snd <$> sublistN n keys
 
+encKeys :: [(PublicKey, EncryptedSecretKey, PassPhrase)]
+encKeys =
+    deterministic "enc keys" $
```

Comment:
It's not good for this task, because the space of all keys is fixed and chances of getting a key which leads to small CRC32 are small. In `Test.Pos.Core.AddressSpec` I used slightly different approach.